.github/workflows/docs-ci.yml

```diff
@@ -101,3 +101,27 @@ jobs:
           remote_port: ${{ secrets.DOCS_PORT }}
           remote_user: ${{ secrets.DOCS_USER }}
           remote_key: ${{ secrets.DEEPHAVEN_CORE_SSH_KEY }}
+
+  pyclientdoc:
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Generate the docs in html
+        run: |
+          pip3 install sphinx sphinx-autodoc-typehints
+          cd pyclient/docs
+          make html
```

Comment:
If we are going to rely on the system python (instead of doing it through gradle), I think we'll want to have more control over the python setup (just like we choose the version of java).

We'll want to use https://github.com/actions/setup-python.

Here are some examples https://docs.github.com/en/actions/guides/building-and-testing-python